db/migrate/20170923230117_create_comments.rb

```diff
@@ -0,0 +1,12 @@
+class CreateComments < ActiveRecord::Migration[5.1]
+  def change
+    create_table :comments do |t|
+      t.text :body
+      t.integer :user_id, null: false
+      t.integer :content_id, null: false
+      t.string :content_type, null: false
```

Comment:
Just an FYI, you could have used `t.references` or `t.belongs_to` to achieve the same result for foreign keys.

```ruby
t.references :content, polymorphic: true, null: false
```